pmd-core/src/test/java/net/sourceforge/pmd/renderers/RenderersTests.java

```diff
@@ -15,18 +15,18 @@
  */
 @RunWith(Suite.class)
 @SuiteClasses({
-        CodeClimateRendererTest.class,
-        CSVRendererTest.class,
-        EmacsRendererTest.class,
-        HTMLRendererTest.class,
-        IDEAJRendererTest.class,
-        PapariTextRendererTest.class,
-        SummaryHTMLRendererTest.class,
-        TextPadRendererTest.class,
-        VBHTMLRendererTest.class,
-        XMLRendererTest.class,
-        XSLTRendererTest.class,
-        YAHTMLRendererTest.class
```

Comment:
https://github.com/pmd/build-tools/blob/a847dc52ccdaafff9b8324da4390bc2043f6e926/src/main/resources/net/sourceforge/pmd/pmd-checkstyle-config.xml#L230-L233
In your pmd config the arrayInitIndent, basicOffset, lineWrapping is 4 and thus these lines should be indented at level 8.